client/src/reducers/main.js

```diff
@@ -0,0 +1,18 @@
+/*
+*
+*  This file is for the Main Reducer
+* Must also import the other reducers to this file
+*
+
+import { combineReducers } from 'redux';
+
+const rootReducer = combineReducers({
+  //stateKey1: stateReducer1,
+  //stateKey2: stateReducer2,
+  //stateKey3: stateReducer3,
+  //etc
+});
+
+ export default rootReducer; //exporting to redux store (store.js)
+
+*/
```

Comment:
as before, avoid pushing commented code to GitHub, even as an example
also don't push empty files or files with only comments.

Reply:
Comments are purely temporary to provide an example to those new to redux, once actual files of this nature are ready to be merged to main branch, they can be overwritten or deleted, purely used to provide assistance for the team